src/Backups/BackupImpl.cpp

```diff
@@ -275,7 +275,7 @@ void BackupImpl::writeBackupMetadata()
     assert(!is_internal_backup);
 
     Poco::AutoPtr<Poco::Util::XMLConfiguration> config{new Poco::Util::XMLConfiguration()};
-    config->setUInt("version", CURRENT_BACKUP_VERSION);
+    config->setUInt64("version", CURRENT_BACKUP_VERSION);
```

Comment:
not necessary

Reply:
Yes, but type of this const is UInt64.

Reply:
I'd prefer to make CURRENT_BACKUP_VERSION just int, but that's not important enough anyway

Reply:
I prefer to keep bug fixes as small as possible, but let's convert version to int too.